tools/ci/launch-test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This script builds the project's boot.iso file in the docker environment and
+# next launches a test and checks for a successful timeout.
+set -e
+
+# Execute relative to this script
+SCRIPT="$(readlink -e "$0")"
+declare -r SCRIPT
+SCRIPTDIR="$(dirname "$SCRIPT")"
+declare -r SCRIPTDIR
+
+# Go to root directory of repository
+cd "$SCRIPTDIR"/../..
+
+# Build project in docker
+make clean docker:boot.iso
+
+# Use QEMU to launch the guest
+declare -i STATUS=0
+timeout 10 qemu-system-x86_64 \
+    -cdrom boot.iso \
+    -m 1024 \
+    -no-reboot \
+    -serial stdio \
+    -smp cpus=2 || STATUS=$?
+
+# Check if the expected exit code happened (124 for timeout)
+if [ "$STATUS" -ne 0 ] && [ "$STATUS" -ne 124 ]
```

Comment:
KTF may loop idle instead of power off. That would hit the timeout condition. Maybe grep serial output for "All tests done" to detect success condition?